pylearn2/datasets/dense_design_matrix.py

```diff
@@ -1227,7 +1227,8 @@ def set_topological_view(self, V, axes=('b', 0, 1, 'c'), start=0):
                                             data_x=X,
                                             start=start)
 
-    def init_hdf5(self, path, shapes):
+    @staticmethod
+    def init_hdf5(path, shapes, filters=None):
```

Comment:
The reason is that, I want to make the first time data construction separated from its usage.